R/installation.feedback.R

```diff
@@ -25,24 +25,38 @@ installation.feedback <- function(k, date, snowball, start.time) {
 
 
   # Show feedback
-  msg.header <- paste0("\n\n\ngroundhog.library() says: Installing '", snowball[k, "pkg_vrs"], "', package #", k, " out of ", N, " needed for '", pkg_vrs, "'")
+  msg.header <- paste0("\n\n\ngroundhog.library(), says [using R-", get.rversion(), "]: Installing '", snowball[k, "pkg_vrs"], "', package #", k, " out of ", N, " needed for '", pkg_vrs, "'")
   msg <- paste0(
-    "> The time now is ", format(Sys.time(), "%H:%M"), ", and we quite roughly estimate the ",
-    "process to end around ", finish.time.estimate, "\n",
+    "> The time now is ", format(Sys.time(), "%H:%M"), ", and we quite roughly estimate the",
+    "  process to end around ", finish.time.estimate, "\n",
     "> It is unlikely to finish after ", finish.time.max, "\n",
-    "> These estimates will be revised after each package installs, but they will remain noisy estimates.\n"
+    "> These estimates will be revised after each package installs,\n",
+    "  but they will remain noisy estimates."
   )
 
-  # Add msg if R mismatch
-  if (rv$r.using.major != rv$r.need.major | rv$r.using.minor != rv$r.need.minor) {
+  # Add msg if R mismatch and recent enough for groundhog
+  if ((rv$r.using.major != rv$r.need.major | rv$r.using.minor != rv$r.need.minor) & date > "2014-09-18") {
     msg <- paste0(
       msg,
       "> Installation is slow because you are using R-", get.rversion(), "\n",
       "> If you run this script with the R version available on the date you entered: '", date, "', i.e., R-", get.version("R", date), ",\n",
-      "   the entire installation would take about a minute or two.\n",
-      "> Instructions for running older version of R:  https://grondhogR.com/olderR"
+      "  the entire installation would take about a minute or two.\n",
+      "> Instructions for running older version of R:  http://groundhogR.com/olderR"
```

Comment:
```suggestion
      "> Instructions for running older version of R:  https://groundhogR.com/olderR"
```

Reply:
i usually stick to http:// becuase the server automatically make it https:// when available, and if it in the future SSL certificate expries, users don't get an error message. This is not a communication that the absence of the "s" will prevent people from having

Reply:
From a security standpoint, linking to the http:// version is still insecure even when the server redirects to https:// because the connection can be hijacked before the redirection.

It's not critical here but I still think it's important to use https:// as much as possible. It normalizes it for the users and it's a badge of good practice IMO.

The final choice is yours.

Reply:
ok

Reply:
https:// is fine